nltk/test/tokenize.doctest

```diff
@@ -39,6 +39,9 @@ Some test strings.
     >>> s10 = "There were 300,000, but that wasn't enough."
     >>> word_tokenize(s10)
     ['There', 'were', '300,000', ',', 'but', 'that', 'was', "n't", 'enough', '.']
+    >>> s11 = "«Now that I can do.»"
+    >>> word_tokenize(s11)
+    ['«', 'Now', 'that', 'I', 'can', 'do', '.', '»']
```

Comment:
Similarly the doctest should expect: 

```
[u'\xab', u'Now', u'that', u'I', u'can', u'do', u'.', u'\xbb']
```

But since Python3 doesn't use the `u'somestring'` notation at declaration. You might need to do the test like this:

```
>>> s11 = u'\xabNow that I can do.\xbb'
>>> expected =  [u'\xab', u'Now', u'that', u'I', u'can', u'do', u'.', u'\xbb']
>>> word_tokenize(s11) == expected
True
```

Possibly this example might help you if you encounter problems regarding doctest on both Python2 and Python3: https://github.com/nltk/nltk/blob/develop/nltk/tokenize/toktok.py#L34